### PR TITLE
Feature/cclw updates

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Lots of copy updates
- Bug fixes

### CODE REVIEW

File: frontend/e2e/cypress/support/page_objects/NavigationConstructor.js [hunk 1]
```diff
@@ -59,11 +59,11 @@ export class NavigationConstructor {
     return this;
   }
   methodologyPage() {
-    navigateDropdownMenuItem("a", "Methodology", "/methodology");
+    handleMenuExternalLink("Methodology", "https://github.com/climatepolicyradar/methodology");
```
Comment (diversemix) [inline frontend/e2e/cypress/support/page_objects/NavigationConstructor.js:62]: This link also shows some "Code Book" :shrug: 
Should this actually be: https://github.com/climatepolicyradar/methodology/blob/main/METHODOLOGY.md

File: frontend/src/components/blocks/SearchResult.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const SearchResult = ({ document }: SearchResultProps) => {
             </div>
           </div>
           {document_content_type === "application/pdf" && document_passage_matches.length > 0 && (
-            <button data-docId={document_slug} className="mt-2 lg:mt-0 py-1 px-4 bg-blue-600 text-white font-medium transition duration-300 rounded-lg hover:bg-indigo-600">
+            <button data-slug={document_slug} className="mt-2 lg:mt-0 py-1 px-4 bg-blue-600 text-white font-medium transition duration-300 rounded-lg hover:bg-indigo-600">
```
Comment (diversemix) [inline frontend/src/components/blocks/SearchResult.tsx:45]: oops - I must have missed this - thanks

**Review by diversemix — APPROVED** [review]
lgtm

File: frontend/e2e/cypress/support/page_objects/NavigationConstructor.js (hunk 1 above)
Comment (joel-wright) [reply frontend/e2e/cypress/support/page_objects/NavigationConstructor.js:62]: resolving to just get this merged

**Pull request merged**